rpcserver.go

```diff
@@ -676,17 +681,27 @@ func checkFeeRateSanity(rpcFeeRate uint32) (*chainfee.SatPerKWeight, error) {
 		return nil, fmt.Errorf("manual fee rate below floor: "+
 			"(fee_rate=%d, floor=%d sat/kw)", rpcFeeRate, feeFloor)
 
+	// A manual fee was set but is below minRelayFee.
+	// TODO (gijs):  Can minRelayFee ever be lower than feeFloor? If not, we
```

Comment:
No, that would be a mistake. I can't speak for all backend implementations, whether they would respect that. But if we want to be sure, we'd check `minRelayFee` returned from lnd (which is just passed forward from the chain backend) and check it against the fee flow. If it's below, we just use the fee floor.

Reply:
Resolved it slightly different, so that we can still throw different errors.